### ISSUE TITLE
Auto Network Selection

### ISSUE THREAD

**Issue by s0undt3ch** [CyanogenMod/android_kernel_asus_moorefield#2]
I've been using the build provided on the forum and I'm running it for 3-4 days now.
Everyday I get a notification where the sim network auto selection failed and I have to search and manually select my network.

Please forgive if this is a known issue or not the right channel to submit issues.

Kudos for your work on this ROM!

**Comment by nesak83** [comment]
Yes. I having the same issue here.. even with stable or nightly version..The rest is all ok expect this issue... Excellent ROM !!! Kudos to you guys..
